test-suite/coq-makefile/native4/run.sh

```diff
@@ -25,12 +20,5 @@ sort > desired <<EOT
 ./test/test_plugin.cmxs
 ./test/test.v
 ./test/test.vo
-./test/.coq-native
-./test/.coq-native/Ntest_test.cmi
-./test/.coq-native/Ntest_test.cmx
-./test/.coq-native/Ntest_test.cmxs
```

Comment:
Umm , I'm lost as to what is going on with this test.

Reply:
Wasn't it the point of the test to override the flag?

Reply:
The `native4` test uses `-native-compiler no` (in the _coqproject) so no native products are expected.

Reply:
Yup, but what about the above:
```
# Shoud override the _CoqProject flag "-native-compiler no"
export COQEXTRAFLAGS="-native-compiler yes"
```

Reply:
I guess it doesn't
Let's change that in another pr though

Reply:
I mean the above code was also removed, my question is why?

Reply:
oh I didn't realize
These 4 near-identical tests were annoying to update so I made a mistake.